pkg/apic/apiservicerevision.go

```diff
@@ -95,6 +95,11 @@ func (c *ServiceClient) processRevision(serviceBody *ServiceBody) error {
 	return nil
 }
 
+// GetAPIRevisions - Returns the list of API revisions for the specified filter
+func (c *ServiceClient) GetAPIRevisions(queryParams map[string]string, stage string) ([]v1alpha1.APIServiceRevision, error) {
```

Comment:
Exposing this func so the DA can retrieve all revisions of an api service at startup for caching.